backend/plugins/utilities/textfsm/template.py

```diff
@@ -133,27 +115,45 @@ def insert_template_into_index_lines(self, original_template_lines: typing.List[
 
             elif driver_section_identified and count == 0:  # first line after the last in the right driver section
                 count += 1
-                new_line = f'{template_filename}, .*, {driver}, {command}\n'
+                new_line = f"{template_filename}, .*, {driver}, {command}\n"
                 new_index_lines.append(new_line)
 
             new_index_lines.append(line)
 
         return new_index_lines
 
 
+def return_errors(f):
```

Comment:
I suspect we'll use this elsewhere, and it'll eventually be migrated to a utils module or such.  But it's here for now